app/ports/management/commands/load-rsync.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from ports.models import Port, LastPortIndexUpdate
+from parsing_scripts import rsync_operations
+
+
+class Command(BaseCommand):
+
+    help = "Populates the database with Initial data after fetching portindex.json from rsync"
+
+    def handle(self, *args, **options):
+        if not Port.objects.count() > 0:
+            # Fetch from rsync
+            rsync_operations.sync()
+
+            # Open the file
+            data = rsync_operations.open_file()
+
+            # Start populating
+            Port.load(data['ports'])
+
+            # Write the commit hash into database
+            last_commit = LastPortIndexUpdate.objects.all().first()
+            last_commit.git_commit_hash = data['info']['commit']
+            last_commit.save()
+
+            print("Initial Data Loaded Successfully.")
```

Comment:
It doesn't relate to this line in particular, but I didn't know where to write.

To me the filename `load-rsync` is very misleading. It should be clear that it's about "initial port database population".

Reply:
Okay, thank you. `load-initial-data` could be the name of the file, as well as the function name suggested by you above.

Reply:
I have used `autoload_initial_data` because there is already `load` which needs to supplied a file manually.